test/functional/cmdLineTests/build.xml

```diff
@@ -40,6 +40,21 @@
 		<subant target="">
 			<fileset dir="." includes="*/build.xml" />
 		</subant>
+		<subant target="">
+			<fileset dir="../cmdline_options_tester" includes="build.xml" />
+		</subant>
+		<subant target="">
+			<fileset dir="../cmdline_options_testresources" includes="build.xml" />
+		</subant>
+		<subant target="">
+			<fileset dir="../Java8andUp" includes="build.xml" />
```

Comment:
Java8andUp, JIT_Test, VM_Test are not required for all the cmd line tests. So it's better to have the subant in the specific test build.xml. The other question is if we run make compile for functional level, will these projects compile twice?